main.nf

```diff
@@ -218,19 +217,6 @@ process mask_beds {
     cat $bedfile \
         | bedtools intersect -v -a stdin -b $mask1 -f 0.25 \
         | bedtools intersect -v -a stdin -b $mask2 -f 0.25 > \$MASK_FILE
-
-
-    ## In case grep doesn't find anything it will exit with non-zero exit
```

Comment:
Could have been done with `grep || true` too, instead of mucking about with `errexit`.